docs/quick-install.md

```diff
@@ -1,82 +1,111 @@
-sudo sh /tmp/signalfx-agent.sh --realm YOUR_SIGNALFX_REALM -- YOUR_SIGNALFX_API_TOKEN
-```
+`The SignalFx Agent has been successfully installed.`
 
+`Make sure that your system's time is relatively accurate or else datapoints may not be accepted.`
 
+`The agent's main configuration file is located at /etc/signalfx/agent.yaml.`
 
-#### Windows
+4. (Optional) If you want to override the default user and group names, review your [deployment's README file](https://github.com/signalfx/signalfx-agent/tree/master/deployments) and locate the option to set the user/group ownership for the signalfx-agent service.
 
-Note: This content appears on a SignalFx documentation page and on the **Setup** tab of the Smart Agent tile in the SignalFx UI. The following code to install the current version works only if you are viewing these instructions on the **Setup** tab.  
+5. If your installation succeeds, proceed to the section **Verify Your Installation**. Otherwise, see the section **Troubleshoot Your Installation**.
 
-From the **Setup** tab, copy and paste the following code into your PowerShell window:
+### Install SignalFx Smart Agent on Windows
 
+1. Run Windows PowerShell.
+
+2. To configure Windows execution policy, download the Smart Agent install script, and run it, copy the following commands and paste them into the Windows PowerShell window:
 
 ```sh
-& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"; ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}
+& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString(‘https://dl.signalfx.com/signalfx-agent.ps1’)); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"; ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(“. {$script} $(&{$args} @params)”))}
 ```
 
-The agent will be installed as a Windows service and will log to the Windows Event Log.
+The install script starts the agent as a Windows service that writes messages to the Windows Event Log.
 
+## Verify Your Installation
 
-### Step 2. Confirm your Installation
+1. To verify that you've successfully installed the SignalFx Smart Agent, run the following command in a command line interface application.
 
+**HINT:** Copy the command and paste it into the application window.
 
-1. To confirm your installation, enter the following command on the Linux or Windows command line:
+**For Linux:** Use `terminal` or a similar app 
 
-    ```sh
-    sudo signalfx-agent status
-    ```
+**For Windows:** Use Windows PowerShell
+
+`sudo signalfx-agent status`
```

Comment:
This command is only applicable on Linux.  For Windows, the PowerShell command should be:
`& "\Program Files\SignalFx\SignalFxAgent\bin\signalfx-agent" status`

Reply:
I'll make this change now.